json/mir-common-json/.gitignore

```diff
@@ -0,0 +1,15 @@
+.dub
+docs.json
+__dummy.html
+docs/
+/mir-common-json
+mir-common-json.so
```

Comment:
Why do we have all those files here? Could we use some clean build process that doesn't leave artifacts everywhere around?

Reply:
This are local for my test. I will remove them